webapp/views/partials/includes/banner.jade

```diff
@@ -0,0 +1,15 @@
+nav(class='navbar navbar-inverse navbar-fixed-top')
+    div(class='container-fluid')
+        div(class='navbar-header')
+            button(type='button', class='navbar-toggle collapsed', data-toggle='collapse', data-target='#header-bar')
+                span(class='sr-only') Toggle navigation
+                span(class='icon-bar')
+                span(class='icon-bar')
+            a(class='navbar-brand')
+        div(class='collapse navbar-collapse', id='header-bar')
+            ul(class='nav navbar-nav')
+                li
+                    a(href='#', ng-click='goHome()') Home
```

Comment:
You can directly navigate to states from from html. Use something like ```ui-sref="home"```. See https://github.com/angular-ui/ui-router under "Nested States and Views"